libpurecool/dyson.py

```diff
@@ -52,6 +52,18 @@ def login(self):
         urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
         _LOGGER.debug("Disabling insecure request warnings since "
                       "dyson are using a self signed certificate.")
+        
+        #Must first check account status
+        accountstatus = requests.get(f"https://{self._dyson_api_url}/v1/userregistration/userstatus?country={self._country}&email={self._email}", headers=self._headers, verify=False)
```

Comment:
```suggestion
        accountstatus = requests.get(
            "https://{0}/v1/userregistration/userstatus".format(self._dyson_api_url),
            params={"country": self._country, "email": self._email},
            headers=self._headers,
            verify=False,
        )
```
Using `params` to get URL encode handled. This does not matter for `country` but very important for `email`. Also, don't use f-string since we need to support Python versions below 3.6.

Reply:
Resolved in 6307f4c